cmd/dstask.go

```diff
@@ -512,7 +512,15 @@ func main() {
 		}
 
 		if cmdLine.Cmd == dstask.CMD_ADD {
-			completions = append(completions, "template:")
+			ts := dstask.LoadTasksFromDisk(dstask.NON_RESOLVED_STATUSES)
+			// If statement below is uncommented, but no taskIDs are added to completions
+			// then tab completion will insert space after "template:"
+			//completions = append(completions, "template:")
```

Comment:
As presently implemented "template:" will not autocomplete if there are no tasks within NON_RESOLVED_STATUSES. This make sense as there are no tasks to copy. 

However, if lines 520/522 is uncommented you may want to consider uncommenting this line as well. Otherwise "template:" will not autocomplete if there aren't any existing template tasks. However, note in this case that a space will be inserted after "template:". I'm not sure how to fix this exactly.